pkg/daemon/ceph/client/crush.go

```diff
@@ -153,7 +153,9 @@ func SetCrushMap(context *clusterd.Context, clusterName, compiledMap string) (st
 
 func SetCrushTunables(context *clusterd.Context, clusterName, profile string) (string, error) {
 	args := []string{"osd", "crush", "tunables", profile}
-	buf, err := ExecuteCephCommandPlain(context, clusterName, args)
+	cmd := NewCephCommand(context, clusterName, args)
+	cmd.JsonOutput = false
```

Comment:
sometimes i really miss default arguments in golang... then this option could be passed to `NewCephCommand()`.

Reply:
Yeh, default args would be great. Is this a change request? We can expand the arguments to the constructor, but there are few exceptions to the default value, hence this formulation.

Reply:
Not a change request, just venting about a missing language feature... Agreed that it's better to avoid the extra args in majority of cases.